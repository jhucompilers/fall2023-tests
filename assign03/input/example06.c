int main(void) {
  int arr[10];
  int *p;

  p = arr; // this should be allowed

  *p = 42; // sets the first element to 42

  return 0;
}
