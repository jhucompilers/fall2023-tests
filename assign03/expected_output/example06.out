0|main|function|function () returning int
2|arr|variable|array of 10 x int
2|p|variable|pointer to int
